Prepared syk hsp and p38

levtsid opened the pull request:
<!--
Thank you for opening a contribution to the OpenFE 2024 industry benchmark repository.
Below are a few things we ask you to kindly fill in and self-check before we
can accept your contribution. Please ignore any irrelevant sections.
-->

## Input submission checklist

<!--
If you are submitting prepared input files please indicate if you have
done the following:
-->

* [ x] created a directory for each system based on the [template directory][templates]
* [ x] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [ x] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [ x] added a PDB file with the protein named `protein.pdb`
* [ x] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [ x] did not re-protonate the residues, but left protonation state of the protein unchanged
* [ x] kept cystallographic waters and metals in the PDB file
* [ x] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [ x] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

<!--
Here please add a summary of what changes you have made
-->
## Summary of changes
Prepared fragments hsp single and double ring systems, fragments p38 and merck syk.
<!--
Also please indicate that you are happy to release these materials under the
combined MIT and CCBY-SA 4.0 licenses of this repository
-->
## Licensing agreement
* [ x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

IAlibay reviewed (changes requested):
Thanks @levtsid

### File renames

Could you rename files in the following manner?
- `Preparation.md` files should be named `PREPARATION_DETAILS.md`
- all ligand SDF files should be named `ligands.sdf`
- All protein PDB files should be named `protien.pdb`

### Queries with the ligand selection for SYK ligands

It is unclear to me as to how the choice of ligands was made for the SYK system.
For example, there are a few `flip` ligands that are claimed to have been removed but were listed as the most potent binders in https://github.com/schrodinger/public_binding_free_energy_benchmark/blob/main/21_4_results/ligand_predictions/merck/syk_pkacorr_out.csv

Additionally, the choice of tautomers for CHEMBL3265000 and CHEMBL3265003 isn't clear. My own look at thing seems to indicate that the original study used  `groundstate` and `tautomer1`, although I'll cc @hannahbaumann here to confirm.

hannahbaumann commented:
Yes, from this file (https://github.com/MCompChem/fep-benchmark/blob/master/syk/ligands.sdf) it looks like in the original study for CHEMBL3265000 the negatively charged version was used (groundstate) and for ligand CHEMBL3265003 the other tautomer, tautomer2.

levtsid commented:
Hi @IAlibay 

I've updated the filenames, changed the ligands used (per the instructions in the doc that I must have missed initially, sorry about that), and updated the branch to the latest. Is there anything else that you need updated or changed?

Thanks,
Lev

IAlibay reviewed (commented):
Thanks @levtsid - this looks good. We're just having a brief discussion internally regarding the intermediate ligands in the SYK set. It should be ok, but I'll get back to you in the next day.

hannahbaumann commented on industry_benchmarks/input_structures/prepared_structures/merck/syk/ligands.sdf:
```diff
@@ -0,0 +1,5339 @@
+ 30 59  1  0  0  0  0
+ 31 60  1  0  0  0  0
+ 32 33  1  0  0  0  0
+ 32 34  2  0  0  0  0
```
```suggestion
 32 34  1  0  0  0  0
```

hannahbaumann commented on industry_benchmarks/input_structures/prepared_structures/merck/syk/ligands.sdf:
```diff
@@ -0,0 +1,5339 @@
+ 30 31  2  0  0  0  0
+ 30 59  1  0  0  0  0
+ 31 60  1  0  0  0  0
+ 32 33  1  0  0  0  0
```
```suggestion
 32 33  2  0  0  0  0
```

hannahbaumann commented on industry_benchmarks/input_structures/prepared_structures/merck/syk/ligands.sdf:
```diff
@@ -0,0 +1,5339 @@
+ 41 42  1  0  0  0  0
+ 42 65  1  0  0  0  0
+ 42 66  1  0  0  0  0
+ 42 67  1  0  0  0  0
```
```suggestion
 42 67  1  0  0  0  0
 M  CHG  1  34  -1
```

hannahbaumann reviewed (changes requested):
Thanks @levtsid ! Apologies for not seeing this earlier, but we noticed that ligand `redocked CHEMBL3265000 ground state` has a negative charge where the sulfonamide is, and in this version the formal charge would be put on the carbonyl oxygen, while in the old file from Schindler et al. the charge sits on the nitrogen. I would have thought that the tautomer where the charge is on the nitrogen would be more favorable, but I could be wrong! Please let us know which tautomer you think would be the more favorable one. If the one where the charge is on the oxygen, then no further actions is needed (you can reject my suggestions), if the other tautomer (with the charge on the nitrogen), then you can accept my suggestions and merge them! Please let us know if you have any questions!

levtsid commented:
Thanks @hannahbaumann We had our med-chemists take a look, they said the negative charge on the oxygen is more likely/stable and favorable, and the 3D-structure of the Schrodinger version fits better in the pocket. Did you also have questions/decisions on the intermediates in this set as well? Thanks for helping out with this!

hannahbaumann reviewed (approved):
Thanks @levtsid for checking this, that sounds good! 

Regarding the intermediates, we had discussed that it would be valid to keep them and include them in the calculations. However, this will then afterwards require a small change to the analysis, more specifically would the plotting script not work in this case as it requires experimental data for each ligand. But that is something we can help you with once the calculations are finished!
